server/tests/test_deserialize_viame_csv.py

```diff
@@ -8,20 +8,22 @@
 test_tuple: List[Tuple[list, dict, dict]] = [
     (
         [
-            # all frames in a track must be of the same type, so the type name for 0 will be ignored
+            # all frames in a track must be the same type, so the type name for 0 will be ignored
             "0,1.png,0,884.66,510,1219.66,737.66,1,-1,ignored,0.98",
             "0,2.png,1,111,222,3333,444,1,-1,typestring,0.55",
             "1,1.png,0,747,457,1039,633,1,-1,type2,1",
             # Keypoint testing with HeadTails Line
-            "2,3.png,2,10,50,20,35,1,-1,type3,0.765,(kp) head 22.4534 45.6564,(kp) tail 55.232 22.3445",
+            "2,3.png,2,10,50,20,35,1,-1,type3,0.765,(kp) head 22.4534 45.6564,"
+            "(kp) tail 55.232 22.3445",
```

Comment:
I don't think this is OK style.  Could you either come up with a way to let these string literals span their full width?  I don't even care if you have to disable this rule for this file, this makes the code harder to read IMO.

Reply:
should the content for these tests be moved to `./testutils` and loaded in like for the image-sort.  Then you no longer are worrying about linting and formatting test data?

Reply:
They're paired input/expected, so you'd have to organize them as such. 

I don't feel that strongly about it, but even if you moved these, you still couldn't have str literals longer than 100chars, which as a concept I would like to resolve.